.github/workflows/ossf-scorecard.yml

```diff
@@ -46,6 +46,6 @@ jobs:
         name: SARIF file
         path: results.sarif
         retention-days: 5
-    - uses: github/codeql-action/upload-sarif@ccf74c947955fd1cf117aef6a0e4e66191ef6f61 # v2.13.4
+    - uses: github/codeql-action/upload-sarif@b7cec7526559c32f1616476ff32d17ba4c59b2d6 # v2.13.4
```

Comment:
Hmm, this changed - I think it's just the comment that's incorrect but I'm struggling to find the source of the hash as it's a sub folder within codeql